src/views/datasets/buoy-telemetry/summary.vue

```diff
@@ -182,8 +202,46 @@ import ExternalLink from "../../../components/base/ExternalLink.vue";
 
 const store = inject("store") as BuoyStore;
 
+const generateQuery = (query) => {
+  const queryParams = Object.entries(query)
+    .map(([key, val]) => `${key}=${val}`)
+    .join("&");
+  return `/dataset/${store.name}/dashboard?${queryParams}`;
+};
+
 const endDate = store.maxDateRaw;
 const startDate = new Date(endDate.valueOf() - 5 * 24 * 60 * 60 * 1000);
+const monthDate = new Date(endDate.valueOf() - 30 * 24 * 60 * 60 * 1000);
+
+const scenarios = [
+  {
+    name: "Phospate and Nitrate, past month",
```

Comment:
I'm finding these titles seem more like they aren't for human consumption? Perhaps they should be more like "Phosphate and Nitrate levels in the past month"? With more fluidity in the words

Reply:
Updated these along with the ones on the other buoy pages